README.md

```diff
@@ -118,9 +118,30 @@ The script waits up to 120 secs (4 x 30 seconds) for the database containers to
 TIMEOUT=${TIMEOUT:-30}
 ```
 
-**Update your hosts file:**
+### Name resolution inside the docker network
```

Comment:
The title hierarchy here is broken, is this a new step as before? Or is it a completely separate section?

Reply:
You are right, I made this step 8) of the "Manual Installation" process.